recipe/build.sh

```diff
@@ -42,9 +42,13 @@ cd build_lib
 cmake -D BUILD_LIB=ON -D BUILD_SHARED_LIBS=ON -D BUILD_MPI=ON -D PKG_MPIIO=ON -D LAMMPS_EXCEPTIONS=yes $args ../cmake
 make # -j${NUM_CPUS}
 cp liblammps${SHLIB_EXT}* ../src  # For compatibility with the original make system.
+cp liblammps_pace.* ../src
+cp build-yaml-cpp/libyaml-cpp-pace${SHLIB_EXT}* ../src
```

Comment:
This part should be done automatically in the build script - otherwise the pace library interface is most likely broken upstream.